docs/09-bootstrapping-kubernetes-workers.md

```diff
@@ -185,6 +185,8 @@ Requires=crio.service
 
 [Service]
 ExecStart=/usr/local/bin/kubelet \\
+  --anonymous-auth=false \\
```

Comment:
Do not automatically consider clients to the Kubelet as ```system:anonymous``` in case someone unknowingly makes the ```system:anonymous``` role have actual permissions.